src/Explorer/Panes/PublishNotebookPaneAdapter.tsx

```diff
@@ -146,11 +146,15 @@ export class PublishNotebookPaneAdapter implements ReactAdapter {
 
     let startKey: number;
 
-    try {
-      if (!this.name || !this.description || !this.author) {
-        throw new Error("Name, description, and author are required");
-      }
+    if (!this.name || !this.description || !this.author || !this.imageSrc) {
+      const formError = `Failed to publish ${this.name} to gallery`;
```

Comment:
```suggestion
      const formError = `Failed to publish ${FileSystemUtil.stripExtension(this.name, "ipynb")} to gallery`;
```

Reply:
Btw why can't we just throw like before?

Reply:
the clearPublishingMessage is used to remove the "Publishing to gallery ..." message i suppose. An unintended side effect is that it wipesout the Notification console message for the error that is thrown from the validation. So had to separate it out.